tests/testmacro.nim

```diff
@@ -80,3 +80,48 @@ suite "Macro transformations test suite":
     check waitFor(testAwait()) == true
   test "`awaitne` command test":
     check waitFor(testAwaitne()) == true
+
+suite "Exceptions tracking":
+  test "Can raise valid exception":
```

Comment:
You need some tests with return values (all current tests use `Future[void]`).

Reply:
Neat, thanks! Could be a nice addition to unittest2